src/components/text-editor/prosemirror-adapter/menu/menu-commands.ts

```diff
@@ -138,6 +183,16 @@ const toggleBlockType = (schema, type, attrs = {}, wrap = false) => {
     };
 };
 
+export const isValidUrl = (text: string): boolean => {
```

Comment:
This should probably be the same as "validateLink". But if we use that one I think almost anything will be considered a link 😅